code/datums/dna.dm

```diff
@@ -305,6 +305,11 @@
 		var/datum/species/old_species = dna.species
 		dna.species = new_race
 		dna.species.on_species_gain(src, old_species, pref_load)
+		qdel(language_holder)
+		var/species_holder = initial(mrace.species_language_holder)
+		language_holder = new species_holder(src)
+		if(mind)

```

Comment:
Why the mind check here ?

Reply:
Left over from old, mindless updates are handled in the proc now.